spec/models/automation_request_spec.rb

```diff
@@ -113,13 +113,13 @@
       ar = described_class.create_from_scheduled_task(admin, @uri_parts, @parameters)
       expect(ar).to be_kind_of(AutomationRequest)
       expect(ar).to eq(AutomationRequest.first)
-      expect(ar).to have_attributes(
+      expect(ar.attributes).to include(
```

Comment:
You shouldn't need to call `.attributes` on an `ApplicationRecord` descendant.

Reply:
Errors without it:
```
expected #<AutomationRequest id: 19, description: "Automation Task",..., but it does not respond to `include?`
```

Reply:
Oh, wait... there's nothing wrong with `have_attributes` on `ApplicationRecord` descendants.

Reply:
`Find and Replace` ☹️ .  I'll remove this change.  Thanks